src/App.js

```diff
@@ -1,28 +1,21 @@
 import React from 'react';
-
 import './App.scss';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './components/PostList';
+
+const AllPosts = posts.map(post => ({
```

Comment:
Let's follow JS naming convention
```suggestion
const allPosts = posts.map(post => ({
```